jenkins/debian/debos/scripts/buster-xfstests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#####################################################
+# This script create buster image for xfstests
+#####################################################
+
+set -e
+
+# Install required xfstests packages
+apt-get install --no-install-recommends xfslibs-dev uuid-dev libtool-bin \
+	        e2fsprogs automake gcc libuuid1 quota attr libattr1-dev make \
+	        libacl1-dev libaio-dev xfsprogs libgdbm-dev gawk fio dbench \
+	        uuid-runtime python sqlite3 git bc bsdmainutils acl -y
+
+apt-get install perl hostname net-tools -y
+
+# Clone repo
+git clone git://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git /xfstests
+
+# Setup xfstests accounts, directory
+useradd fsgqa -m
+useradd 123456-fsgqa
+
+mkdir /test /scratch
+
+# Setup default xfstests config
+cat > /xfstests/local.config <<'EOF'
```

Comment:
What if we want to run tests on something else than `/dev/sda`?  I think this should be set at runtime by the LAVA job, with a template parameter define in `test-configs.yaml`.

Reply:
Since partition used inside rootfs image itself this remained static. Now we are moving partition out of 'rootfs' image, yes , it should be set at runtime. Will check and add it  `test-configs.yaml`

Reply:
I'm unable figure out this.  I looked into `test-configs.yaml` I assume, we will add something like `fstype: 'ext4'` which will be used in `templates/xfstests/file.jinja2` which runs the job?

Reply:
You can use the `params` attribute in a test definition, this is a dictionary passed to the template.